source/System.cpp

```diff
@@ -174,7 +174,7 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 
 		// Handle the attributes which can be "removed."
 		if(key == "hidden")
-			hidden = !remove;
+			hidden = true;
```

Comment:
I changed this from `hidden = true` to `hidden = !remove` a while back but then realized it was unnecessary, so flipping it the other away again while we're here.